Shadowrun6th-German/translations/fr.json

```diff
@@ -1,364 +1,364 @@
+	"holdout": "Holdout-Pistol",
+	"hide": "Caché?",
+	"illusion_air": "Illusion (Air)",
+	"illusion_beast": "Illusion (Bête)",
+	"illusion_blood": "Illusion (Sang)",
+	"illusion_earth": "Illusion (Terre)",
+	"illusion_fire": "Illusion (Feu)",
+	"illusion_guardian": "Illusion (Gardien)",
+	"illusion_man": "Illusion (Homme)",
+	"illusion_plant": "Illusion (Plante)",
+	"illusion_task": "Illusion (Tâche)",
+	"illusion_water": "Illusion (Eau)",
+	"ini": "Initiative",
+	"iniastral": "Initiative (Astral",
+	"inidice": "Dé d'Initiative",
+	"iniphys": "Initiative (physique)",
+	"inimatrix": "Initiative (Hot Sim)",
 	"initiative": "Initiative",
 	"intuition": "Intuition",
 	"intuition_short": "INT",
-	"int_wil": "INT + WIL",
-	"judgeintent": "Menschenkenntnis",
-	"lightpistol": "Leichte Pistole",
-	"liftcarry": "Heben & Tragen",
-	"logic": "Logik",
+	"int_wil": "INT + VOL",
+	"judgeintent": "Juger les intentions",
+	"lightpistol": "Pistolet léger",
+	"liftcarry": "Porter & Soulever",
+	"logic": "Logic",
 	"logic_short": "LOG",
 	"log_int": "LOG + INT",
-	"machinegun": "Maschinengewehr",
-	"machinepistol": "Automatikpistole",
+	"machinegun": "Mitrailette",
+	"machinepistol": "Pistolet Automatique",
 	"magic": "Magie",
 	"magic_short": "MAG",
-	"man": "Mensch",
-	"manipulation_air": "Manip. (Luft)",
+	"man": "Homme",
+	"manipulation_air": "Manip. (Air)",
 	"manipulation_beast": "Manip. (Tier)",
 	"manipulation_blood": "Manip. (Blut)",
-	"manipulation_earth": "Manip. (Erde)",
-	"manipulation_fire": "Manip. (Feuer)",
-	"manipulation_guardian": "Manip. (Beschützer)",
-	"manipulation_man": "Manip. (Mensch)",
-	"manipulation_plant": "Manip. (Pflanze)",
-	"manipulation_task": "Manip. (Helfer)",
-	"manipulation_water": "Manip. (Wasser)",
-	"matrix": "Matrix",
-	"memory": "Erinnerungsvermögen",
-	"mod": "Mod",
-	"mod_conjuring": "Allgemeiner Beschwörenmodifikator",
-	"mod_magic": "Allgemeiner Magiemodifikator",
-	"mod_spells": "Allgemeiner Zaubereimodifikator",
-	"monitor_modifier_phys": "Modifikator körperlicher Monitor",
-	"monitor_modifier_stun": "Modifikator geistiger Monitor",
-	"noedge": "Du hast kein Edge.",
-	"notes": "Notizen",
+	"manipulation_earth": "Manip. (Terre)",
+	"manipulation_fire": "Manip. (Feu)",
+	"manipulation_guardian": "Manip. (Gardien)",
+	"manipulation_man": "Manip. (Hommes)",
+	"manipulation_plant": "Manip. (Plante)",
+	"manipulation_task": "Manip. (Tâche)",
+	"manipulation_water": "Manip. (Eau)",
+	"matrix": "Matrice",
+	"memory": "Mémoire",
+	"mod": "Modificateurs",
+	"mod_conjuring": "Modificateurs de conjuration",
+	"mod_magic": "Modificateurs de magie",
+	"mod_spells": "Modificateurs de sorcellerie",
+	"monitor_modifier_phys": "Modificateur de moniteur de condition physiques",
+	"monitor_modifier_stun": "Modificateur de moniteur de condition d'étourdissement",
+	"noedge": "Tu n'as plus de Chance/Atout.",
+	"notes": "Notes",
 	"offense": "Angriffswurf",
 	"phys": "körperlich",
 	"phys_short": "K",
```

Comment:
If that's the short for physical damage we need a P.

Reply:
@Bltgn nice catch! It's fixed now :)